k8s/sandbox/common/camunda/camunda-bpm.yaml

```diff
@@ -0,0 +1,44 @@
+---
+apiVersion: flux.weave.works/v1beta1
+kind: HelmRelease
+metadata:
+  name: camunda-bpm
+  namespace: camunda
+  annotations:
+    flux.weave.works/automated: "false"
+spec:
+  releaseName: camunda-bpm
+  chart:
+    repository: https://hmcts.azurecr.io/helm/v1/repo/
+    name: camunda
+    version: 1.1.0
+  values:
```

Comment:
most of these values should be set in the chart.
probe, application port etc...